index.js

```diff
@@ -15,16 +29,16 @@ const aliased = [
   'utils',
 ];
 
-function isInternalModule({ moduleName }) {
-  return moduleName.startsWith('@qsi/');
+function isScopedModule({ moduleName }) {
```

Comment:
I figured it probably wouldn't be a big deal, I can't make the stuff configurable, this was sort of the middle ground we chatted about in #2